modules/insights-advisor-recommendations-filters.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="insights-operator-advisor-recommendation-filters_{context}"]
+= Advisor recommendation filters
+
+The Insights advisor service can return a large number of recommendations. To focus on your most critical recommendations, you can apply filters to the https://console.redhat.com/openshift/insights/advisor/recommendations[Advisor recommendations] list to remove low-priority recommendations.
+
+By default, filters are set to only show enabled recommendations that are impacting one or more clusters. To view all or disabled recommendations in the Insights library, you can customize the filters.
+
+To apply a filter, select a filter type and then set its value based on the options that are available in the drop-down list. You can apply multiple filters to the list of recommendations.
+
+You can set the following filter types:
+
```

Comment:
Not sure about this, but maybe the items here would be easier to read at a glance if they were in a table rather than a bulleted list? WDYT?

In any case, per [the ISG guidance on the wording of list items](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#wording-of-list-items), all of the first sentences of the descriptions must be made grammatically parallel. Re-write so that either they all begin with an imperative "Search for...", "Filter by.." to form complete sentences or they begin with a descriptive sentence fragment (as in the "Total risk" item).

Reply:
done